extra/man/alacritty.scd

```diff
@@ -0,0 +1,92 @@
+ALACRITTY(1)
+
+# NAME
+
+Alacritty - A fast, cross-platform, OpenGL terminal emulator
+
+# SYNOPSIS
+
+Alacritty is a modern terminal emulator that comes with sensible defaults, but
+allows for extensive configuration. By integrating with other applications,
+rather than reimplementing their functionality, it manages to provide a flexible
+set of features with high performance.
+
+# FLAGS
+
+*-h, --help*
+	Prints help information
+
+*--hold*
+	Remain open after child process exits
+
+*--print-events*
+	Print all events to stdout
+
+*-q*
+	Reduces the level of verbosity (the min level is *-qq*)
+
+*--ref-test*
+	Generates ref test
+
+*-v*
+	Increases the level of verbosity (the max level is *-vvv*)
+
+*-V, --version*
+	Prints version information
+
+# OPTIONS
+
+*--class* <general> | <general>,<instance>
+	Defines the window class hint on Linux [default: Alacritty,Alacritty]
+
+	When only the general class is passed, instance will be set to the same value.
+
+	On Wayland the general class sets the `app_id`, while the instance class is ignored.
+
+*-e, --command* <command>...
+	Command and args to execute (must be last argument)
+
+*--config-file* <config-file>
+	Specify alternative configuration file
+
+	Alacritty looks for the configuration file at the following paths:
+		. $XDG_CONFIG_HOME/alacritty/alacritty.yml
+		. $XDG_CONFIG_HOME/alacritty.yml
+		. $HOME/.config/alacritty/alacritty.yml
+		. $HOME/.alacritty.yml
```

Comment:
```suggestion
		- $XDG_CONFIG_HOME/alacritty/alacritty.yml
		- $XDG_CONFIG_HOME/alacritty.yml
		- $HOME/.config/alacritty/alacritty.yml
		- $HOME/.alacritty.yml
```

Reply:
Why do you suggest this change? This was an ordered list before so this is intentional.

Reply:
I simply suggest to use `-` to form a character you want anyway. Nothing will be changed other than using bigger `dot`.

Reply:
Haven't seen the numbers generated.